common/src/main/java/com/wynntils/features/combat/HorseMountFeature.java

```diff
@@ -60,19 +61,20 @@ public void onUseItem(UseItemEvent event) {
     private void mountHorse() {
         if (!Models.WorldState.onWorld()) return;
 
-        if (McUtils.player().getVehicle() != null) {
+        LocalPlayer player = McUtils.player();
```

Comment:
add a player null check here?

Reply:
That won't be necessary. I checked the access paths to this point and it can only end up here in active game play. The problem were, as I said, in the tick delayed code.